Dockerfile

```diff
@@ -32,9 +33,9 @@ ENV     NODE_PATH /calypso/server:/calypso/client
 
 # Install base npm packages to take advantage of the docker cache
 COPY    ./package.json /calypso/package.json
-COPY    ./npm-shrinkwrap.json /calypso/npm-shrinkwrap.json
-# Sometimes "npm install" fails the first time when the cache is empty, so we retry once if it failed
-RUN     npm install --production || npm install --production
+COPY    ./yarn.lock /calypso/yarn.lock
+
+RUN     curl -o- -L https://yarnpkg.com/install.sh | bash && export PATH="$HOME/.yarn/bin:$PATH" && yarn install
```

Comment:
> yarn install --production to skip over devDependencies

probably not @gwwar because the Docker container builds the bundles from source. we may not need linting but we certainly need the loaders and babel etc…